taskcluster/app_services_taskgraph/transforms/module_build.py

```diff
@@ -12,6 +12,25 @@
 transforms = TransformSequence()
 
 
+@transforms.add
+def release_upload_symbols(config, tasks):
+    for task in tasks:
+        if config.params["tasks_for"] == u"github-release":
+            if task["attributes"]["buildconfig"]["uploadSymbols"]:
+                task["run"].setdefault("post-gradlew", [])
+                task["run"]["post-gradlew"].append(
+                    [
+                        "source",
+                        "automation/upload_android_symbols.sh",
+                        "{}".format(
```

Comment:
Nit: Unnecessary `format()`

Reply:
Good catch. Initially I had the last script call and the formatting part in one place but run-script bashing expects it as a list of arguments. Done!